tools/AutoTuner/pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[project]
+name = "autotuner"
+version = "0.0.1"
+description = "This project provides a set of tools for tuning OpenROAD-flow-scripts parameter without user interference."
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: BSD 3-Clause",
+]
+readme = "README.md"
+dependencies = [
```

Comment:
potentially consider the separation of requirements and requirements-test

```
[tool.setuptools.dynamic]
dependencies = { file = ["requirements.txt"] }
optional-dependencies = { dev = { file = ["requirements-test.txt"] } }
```

Reply:
Great idea, but are there any test-specific dependencies? From what I've saw only additional dependency in tests is `unittest`, which is Python built-in module.

Also, is there a specific reason to store dependencies in separate requirements files instead of `pyproject.toml` (which is default way I believe)?

Reply:
No particular reason. Just that we want to make it compatible with package locking in the future. (i.e. pip-compile/ poetry etc)

Reply:
Sure, I moved dependencies to separate files - for now `requirements-dev.txt` have only `black` formatter.